index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Home
+order: 0
+main_link: True
+---
+
+# About mozregression
+
+mozregression is an interactive regression range finder for Mozilla *nightly*
+and *inbound* builds. It uses a binary search algorithm for quickly determining
+a changeset range corresponding to when a problem was introduced. It is widely
+used by Mozilla developers and other community members to help find regressions.
+
+Currently, mozregression can works with:
+
+{% comment %}I am not able to write it with line breaks and keep good list formatting{% endcomment %}
+{% for app in site.data.mozregression_supported_apps %}{% assign first = site.data.mozregression_supported_apps | first %}
```

Comment:
Would it be possible to link to the relevant projects? People might not know what B2G is. For B2G, please add a paranthetical comment indicating that it refers to FirefoxOS, and only works with the desktop version.